recipes/nanopb/all/conanfile.py

```diff
@@ -0,0 +1,46 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir, replace_in_file
+from os import path
+
+class NanoPbConan(ConanFile):
+    name = "nanopb"
+    version = "0.4.8"
+    license = "zlib"
+    url = "https://jpa.kapsi.fi/nanopb/"
+    description = "Protocol Buffers with small code size"
+    settings = "os", "compiler", "build_type", "arch"
+    exports = '*'
+    generators = "CMakeToolchain", "CMakeDeps"
+    options = {
+        "Shared": [True, False], "fPIC": [True, False],
+    }
+    default_options = {
+        "Shared": False, "fPIC": True,
+    }
+
+    tool_requires = "protobuf/3.21.9"
+
+    def layout(self):
+        cmake_layout(self)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def configure(self):
+        if self.settings.os == "Windows" and self.settings.compiler == "Visual Studio":
+            del self.options.fPIC
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure(source_folder=path.join(self.source_folder, "conan-wrapper"))
```

Comment:
```suggestion
        cmake.configure()
```
while the project itself has a conan wrapper CMakeLists.txt we don't really need it.